site/app/templates/course/UploadCourseMaterialsForm.twig

```diff
@@ -15,6 +15,36 @@
             </p>
             <input type="text" id="input-provide-full-path" class="full-width" name="eg_config_path" value="" placeholder="optional/subdirectory"/>
         </label>
+        <label id="upload_sections">
+            <p>
+                <strong>Do you want to restrict this course material to some sections?</strong><br />
+            </p>
+            <label class = "radio">
+                <input id = "all_Sections_Showing" name="show_Some_Section_Selection" type = "radio" onchange="showDiv(0)" checked>
+                No
+            </label>
+            <label class = "radio">
+                <input name="show_Some_Section_Selection" type = "radio" onchange="showDiv(1)">
+                Yes
+            </label>
+            <script type="text/javascript">
+                function showDiv(val){
+                    if(val==1){
+                        document.getElementById('show_Some_Section_Selection').style.display = "block";
+                    } else{
+                        document.getElementById('show_Some_Section_Selection').style.display = "none";
+                    }
+                } 
+            </script>
+            <div id = "show_Some_Section_Selection" style="display: none">
+            {% for section in reg_sections %}
+                <tr>
+                    <td>Section {{ section['sections_registration_id'] }}</td>
+                    <td><input id="section_{{ section['sections_registration_id'] }}" type="checkbox" value="off"/></td>
```

Comment:
HTML IDs should use hyphens instead of underscores 
https://submitty.org/developer/coding_style_guide/css#styling

Reply:
Changed section_{{ section['sections_registration_id'] to section-{{ section['sections_registration_id']